libs/ui-components/src/components/Device/DeviceDetails/DeviceDetailsPage.tsx

```diff
@@ -106,7 +106,7 @@ const DeviceDetailsPage = ({ children, hideTerminal }: DeviceDetailsPageProps) =
           </DetailsPageActions>
         ) : (
           canDelete && (
-            <Button aria-label={t('Delete device forever')} variant="danger">
+            <Button component="a" aria-label={t('Delete device forever')} variant="danger" tabIndex={0}>
```

Comment:
should we use `variant='link'` instead ?

Reply:
According to the design, it should use the `danger` variant.